en/features_mc/throw_launch.md

```diff
@@ -0,0 +1,87 @@
+# Throw Launch (Multicopter)
+
+:::warning Experimental
+This feature was introduced after PX4 v1.14.
+
+- It has not yet been broadly tested on different vehicle configurations or scenarios. various platforms and in different scenarios.
+- The majority of testing has been done in position mode.
+  Other modes should also work.
+
+:::
+
+This feature allows a multicopter to be started by arming it from a fixed position and then throwing it into the air.
+The vehicle then turns on the motors and operates according to its current mode.
+
+The vehicle detects that it has been thrown based on reaching a certain speed and altitude (5m/s and 2m by default, respectively) and then starts the motors at the apex of the throw, once it determines that it has started to fall.
```

Comment:
Actually, altitude is not checked, only the speed. I've mentioned that the drone should be thrown "about 2m away and 2m up" just as an example of a throw that results in a speed of 5m/s. Do you think it's not clear that was only an example and should be clarified?

Reply:
@zeroos Absolutely - because I read it as a hard limit.
The point though is that we want to throw fast enough to invoke the throw trigger, and high enough that the motors can start before the vehicle falls to the height of a person.
Is 2 m likely to be high enough in most cases?

Reply:
It depends on a platform, but I feel like for most multicopters small enough to be thrown, it should be fine, unless they are very limited in power. I'll try to make it clear in the modification.

Reply:
No need, I have fixed it :-)